code/game/objects/structures/spirit_board.dm

```diff
@@ -6,77 +6,102 @@
 	resistance_flags = FLAMMABLE
 	density = TRUE
 	anchored = FALSE
+	/// Whether no one has moved the planchette yet.
 	var/virgin = TRUE //applies especially to admins
-	var/next_use = 0
-	var/planchette = "A"
+	/// How long between planchette movements.
+	COOLDOWN_DECLARE(next_use)
+	/// Where the planchette is currently pointing.
+	var/planchette
+	/// Ckey of last mob to use the board.
 	var/lastuser = null
+	/// List of options ghosts (or people) can pick from.
+	var/list/ghosty_options = list(
+		"A","B","C","D","E","F","G","H","I","J","K","L","M","N","O","P","Q","R","S","T","U","V","W","X","Y","Z",
+		"1","2","3","4","5","6","7","8","9","0",
+		"Yes","No",
+	)
+	/// Admin only var, allows admins to bypass the check requiring multiple players
+	var/allow_oneman_use = FALSE
```

Comment:
Perhaps instead of a boolean, it should be an integer for number of people required?